apps/financial-aid/web-osk/src/routes/application/Confirmation/confirmation.tsx

```diff
@@ -22,7 +22,9 @@ import { AppContext } from '@island.is/financial-aid-web/osk/src/components/AppP
 const Confirmation = () => {
   const router = useRouter()
   const { form } = useContext(FormContext)
-  const { municipality } = useContext(AppContext)
+  const { municipality, user } = useContext(AppContext)
+
+  const applicationId = form.applicationId || user?.currentApplication
```

Comment:
currentApplication is now an id, right? Wondering if we should rename it to `currentApplicationId` ?